flaky/flaky_pytest_plugin.py

```diff
@@ -95,6 +96,14 @@ class FlakyPlugin(_FlakyPlugin):
     def stream(self):
         return self._stream
 
+    @property
+    def flaky_success_report(self):
```

Comment:
docstring. "flaky_success_report" isn't obviously a bool property to enable/disable some output.